libs/designer/src/lib/core/index.ts

```diff
@@ -94,7 +94,13 @@ export { storeStateToUndoRedoHistory, onUndoClick, onRedoClick } from './actions
 export { useCanUndo, useCanRedo } from './state/undoRedo/undoRedoSelectors';
 export { resetDesignerView } from './state/designerView/designerViewSlice';
 export * from './queries/runs';
-export { reloadTemplates, resetStateOnResourceChange, type WorkflowTemplateData } from './actions/bjsworkflow/templates';
+export * from './queries/template';
+export {
+  reloadTemplates,
+  resetStateOnResourceChange,
```

Comment:
where is useExistingWorkflowName? btw we do not need that one for consumption.

Reply:
It's in `./queries/template`

Reply:
Ah ok I'm seeing the consumption logic now, well now we have it exported for if we use it with standard in the future.